src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -26,6 +26,8 @@ def initialize()
         config_param :resend_interval_ms, :integer, :default => 30000
         desc 'timeout in millisecond when connecting to djsonsocket'
         config_param :conn_retry_timeout_ms, :integer, :default => 60000
+        desc 'the field name for the event emit time stamp'
+        config_param :emit_timestamp_name, :string, :default => "EmitTimestamp"
```

Comment:
this is the emit from fluentd input plugin timestamp. this is not the arrival time to mdsd output plugin. This time parameter is passed from input plugin to fluentd, then pass all the way to the output plugin.

---
In reply to: [114448612](https://github.com/Azure/fluentd-plugin-mdsd/pull/28#discussion_r114448612) [](ancestors = 114448612)